how-to-build.md

```diff
@@ -46,110 +42,64 @@ On OS X, Clang is used as the C/C++ compiler by default. Clang is installed
 as part of Xcode. Xcode 7.0 or newer is required, and can be installed via
 the Mac App Store.
 
-Configure, build & test
--------------------------
+## Configure, build & test
 
 Run the following commands to configure, build and test core:
 
-    sh build.sh config
-    sh build.sh build
-    sh build.sh test
-
-
-Building for Android
---------------------
-
-Building for Android required the NDK of version >= r10d installed.
-
-It can be built using the following command:
-
-    sh build.sh build-android
-
-Building for iOS, watchOS and tvOS
-----------------------------------
-
-Realm core may be built for iOS, watchOS and tvOS from an OS X machine.
-See the OS X section of [Prerequisites](#prerequisites) above for information
-about the version of Xcode that is required.
-
-To build for iOS:
-
-    sh build.sh build-ios
-
-This produces the following files and directories:
+    mkdir build-dir # create a build folder
+    cd build-dir
+    cmake ..
+    cmake --build
```

Comment:
`cmake --build` appears to require that it be passed a directory. This should probably be `cmake --build .`.